src/backend/resourcemanager/include/resourcepool.h

```diff
@@ -715,6 +715,7 @@ SimpStringPtr build_segment_status_description(SegStat segstat);
 #define EPSILON 1e-7
 #define IS_DOUBLE_ZERO(d)       (fabs(d) < EPSILON)
 #define IS_DOUBLE_EQ(x, y)      ((fabs((x) - (y))) <= (EPSILON))
+#define IS_DOUBLE_EQGREATER(x, y)  (((x) + EPSILON) >= (y))
```

Comment:
or IS_DOUBLE_GE (greater than or equal to)

some examples of abbreviation for comparison operator:
http://folk.uio.no/hpl/scripting/doc/f77/tutorial/logical.html
https://cn.perlmaven.com/comparing-scalars-in-perl
http://www.computerperformance.co.uk/powershell/powershell_comparison_operators.htm